packages/grpc/src/internal/client.ts

```diff
@@ -75,7 +76,8 @@ class ClientBase implements IRequireInitialization, IDisposable {
 
 export function createGrpcClient<T>(
     config: IGrpcClientConfiguration & IGrpcConfiguration,
-    certPath?: string
+    certPath?: string,
+    apiKey?: string
```

Comment:
May be we can turn this into object while this version is still in beta.
option?: {
```suggestion
options?:  {  certPath?: string,
    apiKey?: string
    }
```

Reply:
Created a separate interfaces for server and client